Sources/Amplitude/Utilities/HttpClient.swift

```diff
@@ -72,8 +72,9 @@ class HttpClient {
 
     func getRequestData(events: String) -> Data? {
         let apiKey = configuration.apiKey
+        let clientUploadTime: String = ISO8601DateFormatter().string(from: Date())
```

Comment:
1. I added debug logging and it looks like it doesn't work for some reason:

Debug output:
![image](https://github.com/amplitude/Amplitude-Swift/assets/9509929/f1c36fa2-c919-4728-8a9e-3696b3e7ae94)

User Activity:
![image](https://github.com/amplitude/Amplitude-Swift/assets/9509929/8c7cdca9-86e1-4aec-90db-eed7b59c0706)
Raw Requests:
![image](https://github.com/amplitude/Amplitude-Swift/assets/9509929/2f7c5467-ba75-423f-be81-c6def36f16a4)

2. Note a couple of differences between datetime formats: User Activity and Raw Requests include milliseconds and do not include `T` delimiter and `Z` time zone - maybe it matters.

Reply:
Thank you Andrey for the debugging. The event server wasn't ready until last Friday. I document test with Browser SDK [in this ticket](https://amplitude.atlassian.net/browse/AMP-87291). It should work for Swift SDK too now. I added fractional seconds to Swift SDK so now it's the same format as Browser SDK.